alacritty_terminal/src/selection.rs

```diff
@@ -421,42 +421,6 @@ mod tests {
         Term::new(&MockConfig::default(), size, Mock)
     }
 
-    /// Test case of single cell selection.
-    ///
-    /// 1. [  ]
-    /// 2. [B ]
-    /// 3. [BE]
-    #[test]
-    fn single_cell_left_to_right() {
-        let location = Point { line: 0, col: Column(0) };
-        let mut selection = Selection::new(SelectionType::Simple, location, Side::Left);
-        selection.update(location, Side::Right);
-
-        assert_eq!(selection.to_range(&term(1, 1)).unwrap(), SelectionRange {
-            start: location,
-            end: location,
-            is_block: false
-        });
-    }
-
-    /// Test case of single cell selection.
-    ///
-    /// 1. [  ]
-    /// 2. [ B]
-    /// 3. [EB]
-    #[test]
-    fn single_cell_right_to_left() {
-        let location = Point { line: 0, col: Column(0) };
-        let mut selection = Selection::new(SelectionType::Simple, location, Side::Right);
-        selection.update(location, Side::Left);
-
-        assert_eq!(selection.to_range(&term(1, 1)).unwrap(), SelectionRange {
-            start: location,
-            end: location,
-            is_block: false
-        });
-    }
```

Comment:
Why?

Reply:
Those tests are for the behavior of term when it has only a single cell, and we no longer support that.

Reply:
They're not.

Reply:
Then what are they doing?

Reply:
They're doing exactly what they say they're doing and must not be removed.